client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -58,6 +57,14 @@ const helpLinks = {
 
 const PlansSetup = React.createClass( {
 	displayName: 'PlanSetup',
+	sentTracks: false,
+
+	sendTracks( eventName, options = null ) {
```

Comment:
Can we change this method name for something less generic? something like "trackConfigResults" or something in that line? With "sendTracks" it looks if it were a wrapper over the generic analytics module used in this file to register tracks events, and it's not always the case (https://github.com/Automattic/wp-calypso/pull/6690/files#diff-dd68b4cc9f9bd88ff8d710484a7b86e6R148)